### DISCUSSION TITLE
Ignorance score (IGN) formula in User Guide

### DISCUSSION THREAD

**Discussion by RogerHar** [dtcenter/METplus#2063] [Statistical Computation] [answered]
I just noticed that the formula for the [ignorance score in Appendix C of the MET User Guide](https://met.readthedocs.io/en/latest/Users_Guide/appendixC.html#ign) lacks a factor of 1/2 in the second term when compared to equation 8 of [Gneiting et al 2005](https://journals.ametsoc.org/view/journals/mwre/133/5/mwr2904.1.xml#d5129575e419). I found the corresponding code in [pair_data_ensemble.cc L2036](https://github.com/dtcenter/MET/blob/main_v11.0/src/libcode/vx_statistics/pair_data_ensemble.cc#L2036) and it's correct there, so it looks like it's just the documentation that needs to be fixed.

**Comment by JohnHalleyGotway** [comment, marked as answer]
@RogerHar, wow, excellent catch! Thanks for letting us know. I just updated it in both the [main_v11.0](https://github.com/dtcenter/MET/commit/a71ebc011415f1475f49bcae2c1b1685bbf32f0c) and [develop](https://github.com/dtcenter/MET/commit/f0a44092e448a24a7408cf17be73f7332a769981) branches so that it'll be fixed in all our future releases.

And you should now see it fixed at this RTD location:
https://met.readthedocs.io/en/latest/Users_Guide/appendixC.html#ign

Please let us know if/when you run across other issues.
